### DISCUSSION TITLE
remove footer

### DISCUSSION THREAD

**Discussion by Gordolin** [hbstack/.github#39] [Q&A] [answered]
Hi, your new theme looks realy very good to me, but it is realy complex.  For a private project i do not need any footer. But i can't figure out, where the footer ist linked to the theme. Could you help me?

**Comment by razonyang** [comment, marked as answer]
The easiest way is to override the footer partial. To do this, create `layouts/partials/hb/modules/footer/index.html` with empty content on your site root.
